app/assets/stylesheets/all/option_sets.css.scss

```diff
@@ -109,28 +103,4 @@ form.option_set_form {
    (not inside the form since it's appended to the body) */
 
 div.modal.edit-named-item {
```

Comment:
I've removed this. The naming edit-named-item may also need to be removed from the template. And the modal styling done on the group modal may need to be generally applied to all forms inside modals.